src/modules/esl-carousel/plugin/dots/esl-carousel.nav.dots.ts

```diff
@@ -147,9 +155,9 @@ export class ESLCarouselNavDots extends ESLBaseElement {
   }
 
   /** Updates dots state, rebuilds dots if needed */
-  public update(): void {
+  public update(force?: boolean): void {
     memoize.clear(this, ['count', 'activeIndex']); // invalidate state memoization
-    if (this.$dots.length !== this.count) {
+    if (force || this.$dots.length !== this.count) {
       const $dots = new Array(this.count).fill(null).map((_, index) => this.dotBuilder(index, this));
       $dots.forEach(($dot, index) => $dot.setAttribute('esl-carousel-dot', String(index)));
```

Comment:
Do we need to set esl-carousel-dot attribute?
It seems to me that this operation performs in the dotBuilder method.

Reply:
I've added a comment here, and remove it from default builder
As we agreed: the `esl-carousel-dot` is mandatory system attribute, we do not bother consumer with it but instead making sure it is always presented. (The value of that attribute is also system so user should not rewrite it until he rewrites dots plugin in general)